File: src/lolicon.js
const readline = require('readline').createInterface({
  input: process.stdin,
  output: process.stdout
});

const loli_process = require('child_process');

function intice() {
	// Nothing because weebs are already attracted to underage anime girls
}

readline.question('What\'s the girl\'s age?\n', age => {
  callthefbi(age);
  readline.close();
});

function callthefbi(age) {
	if (age < 15) {
		loli_process.exec('call_911', function callback(error, stdout, stderr){
			console.log(`${error}`);
		});
	} else {
		console.log("Probably just something weird happened.");
	}
}

File: src/lolicon.go
package main

import "os/exec"
import . "fmt"

func intice() {
	// Do nothing because weebs are already attracted to underage anime girls
}

func call_the_fbi (age int) string {
	if (age < 15) {
		cmd := exec.Command("call_911")
		stdout, err := cmd.Output()

		if err != nil {
        	Println(err.Error())
        return ""
    	}

		Print(string(stdout))
	} else {
		return "Probably just something weird happened."
	}
	return ""
}

func main(){
	var age int
	Scan(&age)
	output := call_the_fbi(age)
	Println(output)
}
